department-of-veterans-affairs/va.gov-cms
# VACMS-6188 - Replace media label with rendered entity on node views.

## Description

Closes #6188 .

## Testing done
Viewed a vet center node within local drupal and saw a thumbnail for the media field.

## Screenshots
![image](https://user-images.githubusercontent.com/8375335/138310429-f4a973c3-6414-4773-b830-24c8bf3fc4bb.png)

## QA steps

As a content editor I can view nodes within drupal and see a thumbnail for media fields. 

1. navigate to `admin/content?title=&type=vet_center&moderation_state=All&owner=All`
2. select a vet center such as `Cincinnati Vet Center`
3. view the node and verify that the `Facility photo` field shows an image thumbnail instead of a text link


- [x] No entity types (paragraphs, nodes, blocks, etc) have any view displays that have media fields rendered simply as links to the media entity.
- [ ] Rendered entities mirror the FE experience somewhat (images displayed with appropriate image style, download links displayed as download links
- [ ] Possibly: automated test to ensure that we always use "Rendered entity" or "Thumbnail" display widgets for media fields on node displays.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
